lib/presentation/core/app_widget.dart

```diff
@@ -10,17 +10,19 @@ import 'package:notes_firebase_ddd_course/presentation/sign_in/sign_in_page.dart
 class AppWidget extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
+    final _appRouter = app_router.Router();
```

Comment:
The router should be initialized outside the build(), otherwise, on hot restart it's gonna return an empty screen

Reply:
Yes definitely! Thanks for pointing that out!